scalalib/api/src/mill/scalalib/api/ZincWorkerApi.scala

```diff
@@ -105,7 +108,40 @@ trait ZincWorkerApi {
       scalacPluginClasspath = scalacPluginClasspath,
       reporter = reporter,
       reportCachedProblems = reportCachedProblems,
-      incrementalCompilation = true
+      incrementalCompilation = true,
+      auxiliaryClassFileExtensions = Seq.empty[String]
+    )
+
+  /** Compile a mixed Scala/Java or Scala-only project */
+  @deprecated("Use override with `platformSuffix` parameter", "Mill 0.11.8")
```

Comment:
```suggestion
  @deprecated("Use override with `auxiliaryClassFileExtensions` parameter", "Mill 0.11.8")
```

Reply:
@lolgab Looks like this got lost duering your last force-push.

Reply:
Yeah, messed up with the branches. Pushed it again.